Makefile

```diff
@@ -26,7 +26,13 @@ DOC_FILES := \
 	project.md \
 	media-types.md \
 	manifest.md \
-	serialization.md
+	descriptor.md \
+	image-layout.md \
+	serialization.md \
+	layer.md \
```

Comment:
updated

On Wed, Sep 7, 2016 at 2:49 PM W. Trevor King <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/opencontainers/image-spec/pull/275#discussion_r77881759>
> :
>
> > @@ -26,7 +26,13 @@ DOC_FILES := \
> >  	project.md \
> >  	media-types.md \
> >  	manifest.md \
> > -	serialization.md
> > +	descriptor.md \
> > +	image-layout.md \
> > +	serialization.md \
> > +	layer.md \
>
> This order seems a bit odd to me. I'd rather build this up from the ground
> with something like:
>
>    - …
>    - media-types
>    - descriptor
>    - image-layout
>    - layer
>    - serialization
>    - manifest
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/opencontainers/image-spec/pull/275/files/13755da234e41fb12131570e58514f5ef7ab7866#r77881759>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEF6W6ueT-FRfEkPuksLVLdrzGUO6eKks5qnwcigaJpZM4J3Oq0>
> .
>